### DISCUSSION TITLE
Smarter browser URLs on start/stop

### DISCUSSION THREAD

**Discussion by jgclark** [ttscoff/bunch-docs#111] [Ideas]
I couldn't find an earlier discussion on this; apologies if I missed it.

In some bunches I open a number of URLs. Ideally I'd like to have options (!) for the following:

1. Only open in the browser if it isn't already (applied to the default or specified browser accordingly)
2. Close URL tab when bunch exits, if tab is still open.

Or are either possible already?

**Comment by ttscoff** [comment]
This would be a lot of scripting work to make possible for most browsers, and would be impossible for some browsers (like Firefox).

- Brett

**Comment by ttscoff** [comment]
Websites opened with the display command or other SSB will close with a Bunch, though.

- Brett

**Reply by jgclark** [reply]
> Websites opened with the display command or other SSB will close with a Bunch, though. - Brett

Good point. I'll make an SSB for my frequent use of GitHub , which will help with this.
